test/recipes/30-test_defltfips/fipsmodule.cnf

```diff
@@ -0,0 +1,5 @@
+[fips_sect]
+activate = 1
+conditional-errors = 1
+security-checks = 1
+module-mac = B9:C9:E1:F5:B7:49:18:1B:BF:63:68:DF:1A:66:40:2E:04:2A:8F:E2:B1:D9:F7:7C:08:6F:80:A0:1D:47:F2:00
```

Comment:
There should be a comment that this MAC is deliberately bad.  Perhaps setting it to zeros is also prudent.

I've added a commit making this change.